qiita_pet/templates/sitebase.html

```diff
@@ -133,6 +133,14 @@
                 <li><a href="/study/list/">View Studies</a></li>
               </ul>
             </li>
+            <li class="dropdown">
+              <a href="/user/messages/">Messages</a>
+              <ul class="dropdown-menu">
+              {% for msg, timestamp in current_user.messages(5) %}
```

Comment:
Please, do not perform database accesses in the templates. There has been a fair amount of work to remove any kind of those from the templates. Why did you change this if it was already correct before?